tests/Plaid.Tests/Tests/AssetReportPdfGetAsyncTest.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Going.Plaid.Tests
+{
+	public class AssetReportPdfGetAsyncTest : IClassFixture<PlaidFixture>
+	{
+		private readonly PlaidFixture _fixture;
+		private readonly ITestOutputHelper _output;
+
+		public AssetReportPdfGetAsyncTest(PlaidFixture fixture, ITestOutputHelper output)
+		{
+			_fixture = fixture;
+			_output = output;
+		}
+
+		[Fact]
+		public async Task InvalidAssetReportToken()
+		{
+			var token = "I am very poorly formatted";
+			_fixture.PlaidClient.AccessToken = null;
+			using var pdf = await _fixture.PlaidClient.AssetReportPdfGetAsync(new() { AssetReportToken = token });
+
+			Assert.NotNull(pdf);
+			Assert.NotNull(pdf.Error);
+			Assert.Equal(Entity.ErrorType.InvalidInput, pdf.Error!.ErrorType);
+			Assert.Equal(Entity.ErrorCode.InvalidAssetReportToken, pdf.Error!.ErrorCode);
+		}
+
+		[Fact]
+		public async Task AssetReportPdfGetAsync()
+		{
+			// Step 1. Create the report
+
+			var createrequest = new AssetReport.AssetReportCreateRequest()
+			{
+				AccessTokens = new[] { _fixture.PlaidClient.AccessToken! },
+				DaysRequested = 10,
+			};
+			_fixture.PlaidClient.AccessToken = null;
+			var createresponse = await _fixture.PlaidClient.AssetReportCreateAsync(createrequest);
+
+			Assert.NotNull(createresponse);
+			Assert.True(createresponse.IsSuccessStatusCode, JsonSerializer.Serialize(createresponse));
+
+			var token = createresponse.AssetReportToken;
+			_output.WriteLine($"Token: {token}");
```

Comment:
This will be problematic for me because the test logs for CI builds are public, which means this access token will be public. While it is only for sandbox, it is associated with my account, and nefarious users could use it to disable my account.

Reply:
Not to worry! This is just the token for the REPORT. The worst someone could do is retrieve the report and view your "data". Given this always runs in Sandbox, there should be no actual data leak risk.

I updated the var name to be more clear.

Reply:
Still technically a risk, because if they call the API repeatedly with a token, then Plaid will associate my account with a DOS attack. Prefer to avoid logging secrets of any kind for similar reasons.

Reply:
Ok, no worries, will just drop it out on next update.